### ISSUE TITLE
Prisma Client: Opinionated Subscription implementation

### ISSUE THREAD

**Issue by Weakky** [prisma/prisma#3634]
## Current problem

So far, `client.$subscribe().type()` isn't returning `previousValues` and `node` because they aren't scalars.

## Proposal

Given the following datamodel:

```graphql
type User {
  id: ID! @id
  name: String!
}
``` 

`client.$subscribe.user(...)` should generate the following GraphQL query:

```graphql
subscription {
  user {
    mutation
    updatedFields
    previousValues {
      # ... scalars	(`id`, `name` in this case)	
    }
    node {
	  # ... scalars (`id`, `name` in this case)	
    }
  }
}
```

**Comment by timsuchanek** [comment]
This feature is available in the current beta.

**Comment by pantharshit00** [comment]
This is already available in versions after [1.23.0](https://github.com/prisma/prisma/releases/tag/1.23.0)
